RecFCCeeCalorimeter/src/components/CreateFCCeeCaloNeighbours.cpp

```diff
@@ -51,12 +53,18 @@ StatusCode CreateFCCeeCaloNeighbours::initialize()
   std::pair<int, int> extremaECalLastLayerTheta;
   int ecalBarrelId = -1; // index of ecal barrel in segmentation list
   dd4hep::DDSegmentation::BitFieldCoder *decoderECalBarrel = nullptr;
-  dd4hep::DDSegmentation::FCCSWGridPhiTheta_k4geo *hcalBarrelSegmentation = nullptr;
```

Comment:
sane comment here, the code for the phi-theta grid was removed, should have we kept it and add just the handling of different HCal segmentations?

Reply:
the code for FCCSWGridPhiTheta segmentation is not removed:
https://github.com/Archil-AD/k4RecCalorimeter/blob/main/RecFCCeeCalorimeter/src/components/CreateFCCeeCaloNeighbours.cpp#L234

so it still can be used for this type of  segmentation.